codeforboston/maple
# Navbar mobile refactored design

# Summary

closes #1390 

https://www.figma.com/file/IyRpfsVrU2pg3rysLauvQR/Navigation-V2?type=design&node-id=4356-14&mode=design

# Checklist


# Screenshots

![image](https://github.com/codeforboston/maple/assets/73559781/1950e7c5-2fd1-4eaf-84c9-430e50b9bcc5)

![image](https://github.com/codeforboston/maple/assets/73559781/784f1eb3-e9f9-42d4-834c-b842e89f9412)

![image](https://github.com/codeforboston/maple/assets/73559781/51a13f19-799c-41ec-b4bc-24b6176bdfd6)

![image](https://github.com/codeforboston/maple/assets/73559781/2b992a36-fab4-49f9-a657-6b13723b0ac9)

![image](https://github.com/codeforboston/maple/assets/73559781/32e233a6-a16a-418f-b07c-a198220ae86a)

# Known issues

1) the dropdown menus are grey instead of black in the Figma.  The coloring is pretty hardwired into Bootstrap's theming and is resistant to being overwritten.  Likely, not worth dropping the themeing and doing the entire nav bar's coloration by hand.

2) The Figma wants the expanded Collapse element to fill up the whole screen below the navbar.  For some reason, anytime I tried to set height with either Bootstrap or styled.components, the browser would just ignore that.  I know we are trying to move away from inline styling but in this specific case, it's the only thing the browser seems to respect, so it's one-off exception, barring a better solution being found.

3) like the Desktop view, a General Search function needs to be developed and then can be added to both this and the desktop view

# Steps to test/reproduce

1. make sure your viewport is < 768px wide
2. click on the various navbar elements, including the center logo, and make sure they all respond as expected

![image](https://github.com/codeforboston/maple/assets/73559781/7ac21b66-af40-4237-af22-674766652bf6)
